### ISSUE TITLE
Integrated Console hangs when debugging runspaces

### ISSUE THREAD

**Issue by JustinGrote** [PowerShell/vscode-powershell#3008]
Issue Type: <b>Bug</b>

This has been happening for a while now on insiders/preview. 

When debugging a runspace, if you enter anything at the integrated terminal console, it will hang. Everything else works normally (stepping, debug variables, watch expressions, etc.)

This is probably a PSES item and will xpost the issue there if determined as such.

![Capture](https://user-images.githubusercontent.com/15258962/95903477-13c02580-0d4b-11eb-886f-b5dfdf1ee889.gif)

![image](https://user-images.githubusercontent.com/15258962/95903230-bc21ba00-0d4a-11eb-89f7-638cb54c45bd.png)

Extension version: 2020.9.0
VS Code version: Code - Insiders 1.51.0-insider (605bc4ec1eaf8dc1acd89a0c3c5743926ed0dc09, 2020-10-12T10:52:10.028Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-1035G7 CPU @ 1.20GHz (8 x 1498)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.60GB (1.51GB free)|
|Process Argv|--crash-reporter-id c46712d6-a98d-4728-ab12-732cb7b0c555|
|Screen Reader|no|
|VM|0%|
</details>

**Comment by JustinGrote** [comment]
Related Extension Logs, I don't see a request sent to PSES for the command entered, only for the watch variable which does evaluate correctly.

[ExtensionLogs.zip](https://github.com/PowerShell/vscode-powershell/files/5373518/ExtensionLogs.zip)

```
{"command":"evaluate","arguments":{"expression":"$myvariable","frameId":0,"context":"watch"},"type":"request","seq":9}
10/13/2020 11:55:12 AM [DIAGNOSTIC] - RECEIVED FROM DEBUG ADAPTER: {"seq":11,"type":"response","request_seq":9,"success":true,"command":"evaluate","body":{"result":"\"test\"","variablesReference":0}}
10/13/2020 11:55:12 AM [DIAGNOSTIC] - SENDING TO DEBUG ADAPTER: Content-Length: 72
```

**Comment by rjmholt** [comment]
Probably something we'll fix in PowerShell/PowerShellEditorServices#1295
